Servers/UI/OJS.Servers.Ui/ClientApp/src/hooks/use-urls.tsx

```diff
@@ -44,19 +44,17 @@ interface IUrlsContext {
 const UrlsContext = createContext<IUrlsContext>({} as IUrlsContext);
 
 type IUrlsProviderProps = IHaveChildrenProps
-
-const baseUrl = window.URLS.UI_URL;
-const baseApiUrl = `${baseUrl}/api`;
+const baseApiUrl = `${window.URLS.UI_URL}/api`;
 
 // auth
-const getLoginSubmitUrl = () => `${baseUrl}/Account/Login`;
-const getLogoutUrl = () => `${baseUrl}/Account/Logout`;
+const getLoginSubmitUrl = () => `${window.URLS.UI_URL}/Account/Login`;
+const getLogoutUrl = () => `${window.URLS.UI_URL}/Account/Logout`;
```

Comment:
is this needed? 
It was with baseUrl variable
Can be used as this in Servers/UI/OJS.Servers.Ui/ClientApp/src/utils/urls.ts
```
const
    {
        URLS:
        {
            UI_URL: baseUrl,
            ADMINISTRATION_URL: administrationBaseUrl,
        },
    } = window;
```

Reply:
Fixed.